packages/instant/src/redux/actions.ts

```diff
@@ -25,12 +25,22 @@ export enum ActionTypes {
     UPDATE_SELECTED_ASSET_AMOUNT = 'UPDATE_SELECTED_ASSET_AMOUNT',
     UPDATE_SELECTED_ASSET_BUY_STATE = 'UPDATE_SELECTED_ASSET_BUY_STATE',
     UPDATE_LATEST_BUY_QUOTE = 'UPDATE_LATEST_BUY_QUOTE',
+    UPDATE_BUY_QUOTE_STATE_PENDING = 'UPDATE_BUY_QUOTE_STATE_PENDING',
+    UPDATE_BUY_QUOTE_STATE_FAILURE = 'UPDATE_BUY_QUOTE_STATE_FAILURE',
```

Comment:
`SET_BUY_QUOTE_STATE_PENDING` & `SET_BUY_QUOTE_STATE_FAILURE` ? Update means it takes an argument.

Reply:
Ah, wasn't hip to this convention.  Will change.